File: start/news-reader-android/app/src/main/java/com/example/newsreader/domain/DateFormatter.kt
package com.example.newsreader.domain

import java.text.SimpleDateFormat
import java.util.Date
import java.util.Locale

internal object DateFormatter {
    fun getStringTime(timeInSeconds: Long): String {
        val formatter = SimpleDateFormat("d MMM yyyy", Locale.getDefault())
        return formatter.format(Date(timeInSeconds * 1000))
    }
}

File: start/news-reader-ios/NewsReader/domain/DateFormatter.swift
//
//  DateFormatter.swift
//  NewsReader
//
//  Created by marco.gomiero on 23.11.23.
//

import Foundation


class DateFormatterUtil {
    static func getStringTime(timeInSeconds: Int64) -> String {
        let formatter = DateFormatter()
        formatter.dateFormat = "d MMM yyyy"
        formatter.locale = Locale.current
        return formatter.string(from: Date(timeIntervalSince1970: TimeInterval(timeInSeconds)))
    }
}

